setup.py

```diff
@@ -54,7 +54,7 @@
         'marshmallow==3.0.0b11',
         'scipy',
         'pandas==1.0.5',
```

Comment:
Can we also remove the version lock for Pandas? This has been a problem before when other tools use Ontobio as a dependency.

Reply:
And the lock on networkx too :)

Reply:
oh sure